client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -111,18 +108,13 @@ class SimplePaymentsEdit extends Component {
 					// eslint-disable-next-line
 					console.error( error );
 
+					/*
 					const {
 						data: { key: apiErrorKey },
 					} = error;
-
-					// @TODO errors in other fields
-					this.setState( {
-						fieldEmailError:
-							apiErrorKey === 'spay_email'
-								? sprintf( __( '%s is not a valid email address.' ), email )
-								: null,
-						fieldPriceError: apiErrorKey === 'spay_price' ? __( 'Invalid price.' ) : null,
-					} );
```

Comment:
Noting that removing these is a step backwards but I'm expecting us getting them back soon.